src/show/pages/public/Home/index.jsx

```diff
@@ -1,9 +1,192 @@
+import ConstructorCard from "../../../components/Constructor Card"
+import DriverCard from "../../../components/Driver Card"
+
 const Home = () => {
-<<<<<<< HEAD
-  return <div className="flex ">Home</div>
-=======
-  return <div className="flex">Home</div>
->>>>>>> a86a08de4ca08bdf4ab90e7c4e33d2cebbf81ced
+  return (
+    <div className="flex flex-col w-full h-full bg-primary">
+      <div className="flex bg-charade h-44">
+        <div className="flex ml-5 mt-3 flex-col text-white text-3xl font-bold antialiased">
+          Upcoming Events
+          <div className="flex ml-5 mt-2 w-full h-full ">
+            <img
+              className="flex w-44 h-28"
+              src="https://www.clipartmax.com/png/middle/326-3263821_19-track-vector-race-course-huge-freebie-download-for-formula-1-circuits.png"
+              alt="circuit"
+            ></img>
+            <div className="flex flex-col">
+              <div className="flex ml-5 text-white text-3xl items-center font-bold antialiased">
+                Name of The Event
```

Comment:
Just give it a name, a mock name, temporarily until you will use real data from our backend.